### PR TITLE
Added green boxes

### PR DESCRIPTION
**Pull request by Qlio**
Added 3 green boxes with spaces between

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,11 +8,14 @@
 </head>
 <body>
     <div class="flex">
-        <div class="ulaan"></div>
```
Comment (Qlio) [inline index.html:11]: Энийг солих шаардлага байхгүй.

File: index.css [hunk 2]
```diff
@@ -6,9 +6,16 @@ html,body {
...
     background-color: red;
-}
\ No newline at end of file
+    width: 100px;
+    height: 100px;
+}
+.green {
+    background-color: green;
+    width: 100px;
+    height: 100px;
```
Comment (baska9606) [inline index.css:17]: ene zov uu hha

Comment (Qlio) [reply index.css:17]: zow zow

**Review by Qlio — COMMENTED** [review]
Reviewed

**Pull request merged**